settings.gradle

```diff
@@ -1,23 +1,3 @@
-pluginManagement {
-    repositories {
-        mavenCentral()
-        google()
-    }
-    resolutionStrategy {
-        eachPlugin {
-            if (requested.id.id == "com.android.application") {
-                useModule("com.android.tools.build:gradle:${requested.version}")
-            }
-        }
-    }
-    resolutionStrategy {
-        eachPlugin {
-            if (requested.id.id == "com.android.library") {
-                useModule("com.android.tools.build:gradle:${requested.version}")
-            }
-        }
-    }
-}
```

Comment:
How is this removal related to  the change you're doing?

Reply:
Remove maven repos duplicated in  `buildscript`.